types/typemap.go

```diff
@@ -45,6 +45,8 @@ var typeMap = map[string]interface{}{
 	"handler":                &Handler{},
 	"HandlerSocket":          &HandlerSocket{},
 	"handler_socket":         &HandlerSocket{},
+	"HealthResponse":         &HealthResponse{},
+	"health_response":        &HealthResponse{},
```

Comment:
I'm not sure I understand how this is related to the rest of the PR.

Reply:
This was automatically generated by `go generate`, which was probably not run when `HealthResponse` struct was created!